.azure_pipelines/job_templates/olive-build-doc-template.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
     steps:
     - template: olive-setup-template.yaml
       parameters:
-        python_version: '3.8'
+        python_version: '3.10'
```

Comment:
do the new dependencies require python 3.10? When i tried moving the doc build over to python 3.8, there was an issue with one of the dependencies. it's the same error in the failed doc build ci here